main.css

```diff
@@ -91,21 +101,150 @@ header {
 .main--movie-info {
     display: block;
     position: relative;
-    background-image: url("./images/background.jpg");
     min-height: calc(100vh - 278px);
-    background-size: cover;
-    background-position: 50%;
     z-index: -2;
     color:aliceblue;
 }
 
 .main--movie-info::before {
-    z-index: -1;
     content: "";
     position: absolute;
     width: 100%;
     height: 100%;
     top: 0;
     left: 0;
-    background: rgba(0, 0, 0, 0.7);
+    background: linear-gradient(90deg, #000000, hsl(0deg 0% 0% / 65%))
+}
+
+.error-div {
+    margin: 39.8vh auto;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.movie-container {
+    background-size: cover;
+    background-position: 50%;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    align-items: center;
+    height: 83.6vh;
+}
+
+.movie-container--img {
+    z-index: 1;
+    width: auto;
+    height: 450px;
+    margin: 40px 40px 40px 170px;
+}
+
+.content {
+    z-index: 1;
+    font-size: 1.25rem;
+    font-family: cursive;
+    grid-column: span 3;
+    margin: 0 30px 0 0;
+}
+
+.wrapper-rating {
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.wrapper-rating--rating-text {
+    font-size: 1.5rem;
+}
+
+.rating {
+    display: flex;
+    align-items: center;
+}
+
+.rating-body {
+    width: 150px;
+    height: 60px;
+    position: relative;
+}
+
+.rating-body::before {
+    width: 100%;
+    height: 100%;
+    content: "☆☆☆☆☆";
+    position: absolute;
+    top: 0;
+    left: 0;
+    font-size: 40px;
+    letter-spacing: -4px;
+}
+
+.rating-body__rating-line {
```

Comment:
@Mmarvinn `rating-line` - not sound like a modifier

Reply:
fix